src/frontend/src/lib/services/load-user-profile.services.ts

```diff
@@ -60,10 +60,16 @@ export const loadCertifiedUserProfile = async ({
 };
 
 export const loadUserProfile = async ({
-	identity
+	identity,
+	reload = true
 }: {
 	identity: OptionIdentity;
+	reload?: boolean;
 }): Promise<void> => {
+	if (nonNullish(get(userProfileStore)) && !reload) {
```

Comment:
I was asking myself it we should pass `$userProfileStore` as a parameter or not. Where you hesitating too?